update-helper.js

```diff
@@ -3,9 +3,14 @@ var execSync = require('child_process').execSync;
 var io = require('socket.io-client');
 var inquirer = require('inquirer');
 
+var args = [];
+
 var errorMessage = 'Error, please provide the required updater action: forceupdate | factory | userdata | testmode | cleanupdate | restorevolumio';
-if (process.argv[2]) {
-  var argument = process.argv[2];
+if (process.argv[3]) {
+  var argument = process.argv[3];
+  if (process.argv.length >= 4) {
+    args = process.argv.slice(4);

```

Comment:
we only reassign args array if additional args were passed to the script and assign everything after the `forceupdate` command to that args array